sw/device/lib/dif/dif_warn_unused_result.h

```diff
@@ -5,6 +5,9 @@
 #ifndef OPENTITAN_SW_DEVICE_LIB_DIF_DIF_WARN_UNUSED_RESULT_H_
 #define OPENTITAN_SW_DEVICE_LIB_DIF_DIF_WARN_UNUSED_RESULT_H_
 
+// WARNING: This file will soon be deprecated, since this definition is included
```

Comment:
Just delete the contents of the file and #include the new one.

If we move the macro to macros.h though we can leave this as is (or define DIF_... in terms of the new macro).